content/outbreaks/_index.en.md

```diff
@@ -4,6 +4,94 @@ categories: ["R"]
+
+![Characteristics of cases of acute jaundice syndrome and those testing positive and negative for HEV using RDTs RDTs during an acute outbreak of HEV in Am Timan, Chad, 2016–2017](images/hevperson.png?width=40pc)
+
+**Source**: [Spina A, Lenglet A, et al.(2017) A large outbreak of Hepatitis E virus genotype 1 infection in an urban setting in Chad likely linked to household level transmission factors, 2016-2017. PLoS ONE 12(11): e0188240.](https://doi.org/10.1371/journal.pone.0188240)
+
+## Place characteristics
+
+Here you will create different types of maps to illustrate the geographical spread and impact of the outbreak. Maps can aid in the identification of hotspots for the targeting or improvement of prevention and public health measures.
+
+![Cholera in Yemen, 2017](images/cholerayemen.png?width=40pc)
+
+**Source**: [MSF calls for increased response to cholera outbreak](https://www.msf.ie/article/yemen-msf-calls-increased-response-cholera-outbreak)
+
+# Outbreak templates, the basics
+
+In the Outbreak Template Walkthrough (INSERT LINK), there is a step by step description of how to use the templates for data analysis using outbreak linelists.
+
+We have designed the templates to run through a similar series of data management and analytical questions in the same order:
+
+1. **Import data** and perform initial data cleaning
+2. **Browse and view** your data to get a sense of your dataset structure, naming of variables, etc.
+3. **Re-naming and recoding** variables to help with data analysis (i.e. creation of epidemiological weeks, recoding of age groups, application of case definitions etc.)
+4. Analysis of **personal characteristics**: demographics, signs and symptoms, attack rates, risk groups etc.
+5. Analysis of **temporal characteristics**: creating graphs that display different indicators in time
+6. Analysis of **spatial/geographical characteristics**: creating maps
+
+# Other uses of the templates
+
+As the coding included in the outbreak templates is structured in terms of time, place and person, they can also be used to help guide you through any other analysis for epidemiological data from either:
+
+* outpatient registers
+* hospital registers
+* other surveillance data
+
+You will not be able to use the same set up of the data dictionaries, but you should be able to work out how to recode and rename your variables, create graphs and tables of relevant variables and if necessary generate maps.
+
+There is also coding to be able to guide you on conducting simple statistical comparisons between groups (chi square, t-test etc.), as well as the calculation of Odds Ratios (ORs) and Risk Ratios (RR) and running regression models for multivariate analyses.
+
+> **Below here is all the stuff that was written before that we need to make sure is included in the outbreak template walkthrough**
```

Comment:
Again, adding a TODO so that it's easier to search for these tasks.

```suggestion
> **TODO: Below here is all the stuff that was written before that we need to make sure is included in the outbreak template walkthrough**
```